ScoutSuite/output/data/html/partials/aws/services.ec2.regions.id.vpcs.id.instances.html

```diff
@@ -33,8 +33,23 @@ <h4>Network interfaces</h4>
         <div class="list-group-item" style="padding-bottom: 0 !important;">
             <h4>User data</h4>
             <div>
-                <pre><code>{{#each (split_lines user_data)}}   {{this}}<br>{{/each}}</code></pre>
+                <code>{{#each (split_lines user_data)}}   {{this}}<br>{{/each}}</code>
```

Comment:
Does this still show up correctly in the report? I know I had a lot of trouble displaying this. I don't remember exactly why I put the `<pre>` tag, but I think it was important. hehe

Reply:
The only difference `<pre></pre>` made (that I'm seeing) is remove the code color highlight.